docs/advanced/aml.py

```diff
@@ -0,0 +1,242 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
why there are no tests for this doc? How do we know it actually works?

Reply:
What kind of test should I make

Reply:
what are the options?

Reply:
A unit test or output checking, output checkint doesnt work well since requirements have to be in CI though

Reply:
> output checkint doesnt work well since requirements have to be in CI though

why requirements can't be in CI?